pytrip/let.py

```diff
@@ -49,6 +49,7 @@ class LETCube(Cube):
 
     def __init__(self, cube=None):
         super(LETCube, self).__init__(cube)
+        self.type = "LET"
```

Comment:
What is that ?

Reply:
DosCube has one, so why should LETCube not have one too?

Reply:
I forgot to remove it from initial import, it wasn't used if I am correct.

Reply:
uh, I intended to use it for the GUI

Reply:
why not a class name / type ?

Reply:
```
import pytrip as pt
if isinstance(obj, pytrip.DosCube):
     foobar()
```
vs.
```
if obj.type =='DOS';
    foobar()
```

you can save one line of code, and I just found the latter solution a bit leaner and more convenient, even if less pythonic.

Reply:
lets put that in a new issue